packages/protobuf/src/index.ts

```diff
@@ -5,4 +5,4 @@ export * from './types';
 export { parseConfigure, createMessageFromName, createMessageFromType } from './utils';
 export * as MessagesSchema from './messages-schema';
 // It's problem to reexport enums when they are under MessagesSchema namespace, check packages/connect/src/types/device.ts
-export { DeviceModelInternal } from './messages-schema';
+export { DeviceModelInternal, DisplayRotation, Enum_DisplayRotation } from './messages-schema';
```

Comment:
is this the way we want to export these from connect? cc @martykan

Reply:
I don't know... `DeviceModelInternal` seems important enough to re-export separately, but I don't think it's the best approach for every single enum. 
If we want to re-export in `device.ts` something like this should work:
```
export type DisplayRotation = PROTO.DisplayRotation;
```